iceberg/openhouse/internalcatalog/src/test/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalTableOperationsTest.java

```diff
@@ -76,14 +80,20 @@ void setup() {
     MockitoAnnotations.openMocks(this);
     Mockito.when(mockHouseTableMapper.toHouseTable(Mockito.any(TableMetadata.class)))
         .thenReturn(mockHouseTable);
+    HadoopFileIO fileIO = new HadoopFileIO(new Configuration());
```

Comment:
May be we should add another test for a different FileIO given that HadoopFileIO is the default fileIO.

Reply:
we could add add a test for HadoopFileIO with empty prefix and localFileIO with prefix: `file:`

Reply:
currently added s3 docker tests in testing section. We ca address this in a future PR